src/ng-currency.directive.js

```diff
@@ -1,11 +1,16 @@
 /* @ngInject */
-export default function ngCurrency($filter, $locale) {
+export default function ngCurrency($filter, $locale, ngCurrencySettings) {
   return {
     require: 'ngModel',
     link: (scope, element, attrs, controller) => {
-      let hardCap, min, max, currencySymbol, ngRequired;
       let active = true;
-      let fraction = 2;
+      let currencySymbol, ngRequired;
+      let {
+        defaultHardCap: hardCap,
+        defaultMin: min,
+        defaultMax: max,
+        defaultFraction: fraction
+      } = ngCurrencySettings;
```

Comment:
After default is removed from `ngCurrencySettingsProvider` we can change this to:
```js
let { hardCap, min, max, fraction } = ngCurrencySettings;
```